### PR TITLE
Auto build moderator

### LINKED ISSUE #188: Update moderator model
The model in moderator/moderator_bot appears to be a pickled object, but this no longer works on the latest scikit-learn releases due to a module restructuring. Does anybody know how to rebuild the model? (@jettify?)

### PR DESCRIPTION
**Pull request by Dreamsorcerer**
Fixes #188

The model in the moderator demo needs to be rebuilt to be sure it works correctly with each release of scikit-learn.

The training dataset is ~30MB, so I don't want to commit it to the repo. Additionally, the source file is only accessible after registering on the linked website, and I'm not sure we have copyright permission to redistribute it. So, I've added a URL to the file on my server as a repo secret.

### CODE REVIEW

**Comment by Dreamsorcerer** [comment]
@webknjaz Does this pipeline look right to you?

**Comment by Dreamsorcerer** [comment]
I guess this'll have the same issue of not running the pipeline after commit that we had on aiohttp, but atleast scikit-learn releases are infrequent.

**Comment by webknjaz** [comment]
What do you mean? I see some checks being reported...

**Comment by Dreamsorcerer** [comment]
I meant the diff. I added a post-dependabot job, so it won't run yet.

File: .github/workflows/post-dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        echo "${{ secrets.GITHUB_TOKEN }}" | gh auth login --with-token
+    - name: Checkout
+      run: |
+        gh pr checkout ${{ github.event.pull_request.number }}
+    - name: Setup Python
+      uses: actions/setup-python@v4
+      cache: 'pip'
+      cache-dependency-path: '**/requirements*.txt'
+    - name: Download train.csv
+      run: curl "$MODERATOR_TRAINING_URL" > train.csv
```
Comment (webknjaz) [inline .github/workflows/post-dependabot.yml:35]: What's this secret? Why not have it in the open?

**Comment by webknjaz** [comment]
Ah… Well, use https://github.com/marketplace/actions/github-app-token with `BOT_APP_ID` and `BOT_PRIVATE_KEY` secrets, and it'll output a replacement GitHub token for you.

File: .github/workflows/post-dependabot.yml (hunk 1 above)
Comment (Dreamsorcerer) [reply .github/workflows/post-dependabot.yml:35]: Explained in the PR description.

**Comment by Dreamsorcerer** [comment]
> Ah… Well, use https://github.com/marketplace/actions/github-app-token with `BOT_APP_ID` and `BOT_PRIVATE_KEY` secrets, and it'll output a replacement GitHub token for you.

Like this?
https://github.com/aio-libs/aiohttp-demos/pull/717/commits/907ad17a5a9a73946665b3042eec4937d2de2412

**Comment by webknjaz** [comment]
Yes, that will avoid the limitation of the built-in GHA token.

File: .github/workflows/post-dependabot.yml (hunk 1 above)
Comment (webknjaz) [reply .github/workflows/post-dependabot.yml:44]: Ah, okay. Marking as resolved.

**Pull request merged**